modules/nf-core/sentieon/bwaindex/main.nf

```diff
@@ -0,0 +1,70 @@
+process SENTIEON_BWAINDEX {
+    tag "$fasta"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Sentieon modules does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container 'nfcore/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path(bwa), emit: index
+    path "versions.yml"       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    if [ \${SENTIEON_LICENSE_BASE64:-"unset"} != "unset" ]; then
+        echo "Initializing SENTIEON_LICENSE env variable"
+        LICENSE_ENCODED=\$SENTIEON_LICENSE_BASE64
```

Comment:
Is it really necessary to introduce `LICENSE_ENCODED`? 🤔  Couldn't one make do with just `SENTIEON_LICENSE_BASE64`?

Reply:
good point